packages/app/src/app/pages/Sandbox/Editor/Workspace/items/NotOwnedSandboxInfo/NotOwnedSandboxInfo.tsx

```diff
@@ -2,7 +2,8 @@ import React, { useState } from 'react';
 
 import Dependencies from '../../Dependencies';
 import Files from '../../Files';
-import Project from '../../Project';
+// @ts-ignore
+import Project from '../../Project/index.tsx';
```

Comment:
@SaraVieira No need for the file extension here

```suggestion
import Project from '../../Project/index';
```